api.swagger.yaml

```diff
@@ -6850,6 +7020,212 @@ components:
           type: string
           example: '#FFFFFF'
       type: object
+    group-document-schema:
+      allOf:
+        - type: object
+          properties:
+            uuid:
+              description: 'Invotra UUID of the group document.'
+              type: string
+              format: uuid
+              example: '01234567-89ab-cdef-1234-56789abcdef0'
+        - $ref: '#/components/schemas/group-document-schema-common'
+        - type: object
+          properties:
+            group:
+              description: 'The group for this group document.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/group-schema'
+            image:
+              description: 'Group document Image.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/file-schema'
+            author:
+              description: 'The author of this group document.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/user-trimmed-schema'
+            folder:
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the group document folder.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+                name:
+                  description: 'This is the name of the group document folder.'
+                  type: string
+                  example: 'Darts'
+            timestamp_created:
+              description: 'This is the timestamp (in seconds) of when the group document was created.'
+              type: integer
+              example: 1510941596
+            timestamp_updated:
+              description: 'This is the timestamp (in seconds) of when the group document was last updated.'
+              type: integer
+              example: 1510941696
+            comment_count:
+              description: 'Count of comments on the group document.'
+              type: integer
+              example: 5
+            likes_count:
+              description: 'Count of likes on the group document.'
+              type: integer
+              example: 5
+            published:
+              description: 'Published status of the group document.'
+              type: boolean
+              example: true
+    group-document-schema-update:
+      allOf:
+        - $ref: '#/components/schemas/group-document-schema-common'
+        - type: object
+          properties:
+            group:
+              description: 'The group for this group document.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the group.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            image:
```

Comment:
featured_image

Although there is only one image field on the doc content type, this is a different field in drupal so I think we should name it separately in the api. Just in case the standard image field gets added to doc in the future.

Reply:
done